### ISSUE TITLE
Quick custom color scheme

### ISSUE THREAD

**Issue by evandrocoan** [SublimeText/PackageDev#200]
There is the command `PackageDev: New Color Scheme`, which creates a base structure for a color scheme. But, there is not command for a quick customization of color schemes. 

For example, if I would like to quickly change the comment color, I would call for the command `PackageDev: Custom Current Color Scheme`, then, a new window would open with two files, the first would be the original color scheme as a read only view, the second would be the new file `Current Color Scheme Name.sublime-color.scheme` created on your user directory, initially with bare contents like:
```js
{
  // http://www.sublimetext.com/docs/3/color_schemes.html
  "variables": {
    // "green": "#FF0000",
  },
  "globals": {
    // "foreground": "var(green)",
  },
  "rules": [
    {
      // "scope": "string",
      // "foreground": "#00FF00",
    },
  ]
}
```

___
References:

1. https://forum.sublimetext.com/t/request-ability-to-edit-color-scheme-in-editors-settings/36485/2
1. http://www.sublimetext.com/docs/3/color_schemes.html

**Comment by FichteFoll** [comment]
Sounds useful. I'll add it.

**Comment by FichteFoll** [comment]
Curious about that. If the file exists, it should be opened instead of the placeholder. Even weirder that it seemingly worked on the second try.

**Comment by michaelblyons** [comment]
Can we have a shortcut for [`*.sublime-theme`][1] as well?

[1]: https://www.sublimetext.com/docs/3/dev/themes.html#customization

**Comment by michaelblyons** [comment]
Also, the RHS panel is doing the template version again. The "setup" turns out to be the folder of the file that was open when I ran the command. I've tried it with a couple of files and a couple of different project windows, even new blank unsaved files. It looks for the `color-scheme` in the path of the most recently open saved file, fails to find it, and asks to make the template file there.

![screen shot 2019-01-25 at 5 27 23 pm](https://user-images.githubusercontent.com/2701605/51776442-328a0e80-20c7-11e9-9b2d-f85311644b6e.png)
